HW04.Rmd

```diff
@@ -1,126 +1,260 @@
+         color = "Intent", fill  = "Intent") +
+    theme(panel.background = element_blank(),
+          axis.line = element_line(),
+          plot.title = element_text(hjust = 0.5))
+    
+```
+
+(hint maybe looking at percentages would be good)
+Now use ratio to do another barplot
+```{r race_2}
+ggplot(race_death, aes(x = race, y = ratio, color = intent, fill = intent)) +
+    geom_bar(stat = "identity",
+             position = "dodge") +
+    geom_text(aes(label = round(ratio, 2)), vjust = -1, color = "black", position = position_dodge(width = 0.9)) + ##Any good idea to align these two tags?
+    scale_y_continuous("Ratio",
+                       limits = c(0, 1),
+                       expand = c(0,0)) +
+    xlab("Race") +
+    labs(title = "Ratio of each intent in different races",
+         color = "Intent", fill  = "Intent") +
+    theme(panel.background = element_blank(),
+          axis.line = element_line(),
+          plot.title = element_text(hjust = 0.5))
+```
+From the two barplots above, it seems that whites who are killed by guns more likely to die because of suicide, but the situation is just the opposite for blacks and hispanics.
+
+### Are police-involved gun deaths significantly different from other gun deaths? Assess the relationship between police involvement and age, police involvement and race, and the intersection of all three variables.
+
+```{r police}
+police_death <- gun_death %>%
+    select(police, age, race) %>%
+    arrange(police) %>%
+    filter(age != "Not Available")
+police_death$age <- as.numeric(police_death$age)
+
+    
+```
+
+Think about performing stats such as a t.test to see if differences you are plotting are statistically significant
+
+```{r stats?}
+police_sum1 <- police_death %>%
+    count(police)
+notinv_length <- police_sum1$n[1]
+notinv <- as.integer(police_death$age[1:notinv_length])
+inv <- as.integer(police_death$age[notinv_length+1:length(police_death$age)])
+t.test(x = notinv, y = inv, paired = FALSE)
+```
+Therefore, there is a statistically significant differenct in age of deaths considering the involvement of police.
+
+```{r police_2}
+
+```
+I'm not pretty sure how to analyze the interaction of the three variables...
+```{r stats_2?}
+
+
+```
+
+
+Finally, all together now:
+```{r police_3}
+ggplot(police_death, aes(x = factor(police), y = age, color = race, fill = race)) +
+    geom_bar(stat = "identity",
+             position = "dodge") +
+    scale_x_discrete("Police Involvement",
+                     breaks = c(0, 1),
+                     labels = c("Not involved", "Involved")) +
+    scale_y_continuous("Age",
+                       limits = c(0, 110),
+                       expand = c(0,0)) +
+    labs(title = "Police-involved gun deaths",
+         color = "Race", fill = "Race") +
+    theme(panel.background = element_blank(),
+          axis.line = element_line(),
+          plot.title = element_text(hjust = 0.5))
```

Comment:
What you did here is analyzing the three variables! I also wasn't sure what was the best way to plot these three variables but I did a scatterplot with multiple panels. Ariana recommended doing a boxplot which I think I should have done that but there clearly are many ways to go about this! 

I can't visualize how your graph look like just by reading your code. If you can knit your .Rmd file, that would be really helpful for me to take a look at your graphs instantly without having to copy paste your codes to my RStudio.